patches/minecraft/net/minecraft/commands/CommandSourceStack.java.patch

```diff
@@ -8,4 +8,4 @@
 +public class CommandSourceStack implements SharedSuggestionProvider, net.minecraftforge.common.extensions.IForgeCommandSourceStack {
     public static final SimpleCommandExceptionType f_81286_ = new SimpleCommandExceptionType(Component.m_237115_("permissions.requires.player"));
     public static final SimpleCommandExceptionType f_81287_ = new SimpleCommandExceptionType(Component.m_237115_("permissions.requires.entity"));
-    private final CommandSource f_81288_;
+    public final CommandSource f_81288_;
```

Comment:
This change seems unrelated and should be reverted

Reply:
paging @sciwhiz12  on this, as this was something I was told to leave originally and I believe it was fixed at some point, not sure though.

Reply:
Fixed in #9050